serverFiles/mailer.js

```diff
@@ -0,0 +1,56 @@
+var nodemailer = require('nodemailer');
+
+var transporter = nodemailer.createTransport({
+    service: 'gmail',
+    auth: {
+      user: 'YOUR_EMAIL_HERE@gmail.com',
+      pass: 'YOUR_PASSWORD_HERE'
+    }
+  });
+
+  var sendMail = (userInfo,callback) => {
+
+    console.log(userInfo);
+    var send = `Hi ${userInfo.name}, \n
```

Comment:
This results in formatting with indentation in received email:
![screenshot from 2017-10-29 14-00-26](https://user-images.githubusercontent.com/13132899/32144022-efb9eafe-bcb2-11e7-954e-d6b939b916ff.png)

Please see if you can fix that issue.

Reply:
Yes it's fixed using template string 👍

Reply:
![image](https://user-images.githubusercontent.com/26179770/32144605-3a1323ee-bce1-11e7-87ab-1e8db0e9776c.png)

Fixed 👍